corehq/pillows/case.py

```diff
@@ -72,12 +72,25 @@ def transform_case_for_elasticsearch(doc_dict):
     return doc_ret
 
 
+def transform_case_transactions_to_actions(transactions):
+    def _action(trans):
+        return {
+            'server_date': trans['server_date'],
+            'date': trans['server_date'],
+            'xform_id': trans['form_id'],
+            'sync_log_id': trans['sync_log_id'],
+        }
+    return [_action(trans) for trans in transactions if not trans['revoked']]
```

Comment:
Yea, that's a good point. I think it can be done but I'm don't know if we always want the 'actions' version. I'll take a stab at it.